contents/handbook/growth/marketing/index.md

```diff
@@ -4,19 +4,26 @@ sidebar: Handbook
 showTitle: true
 ---
 
+Marketing & Content exists to drive organic, word-of-mouth growth by:
+
+1. Producing world-class top-of-funnel content for our ICP
+2. Ensuring users can autonomously find answers to their questions
+
```

Comment:
This the only major change. Thoughts on this framing?

Reply:
Pedantic of me, but: 'can autonomously find' feels like it's a discoverability objective and thus may fall more to website and docs, vs something more content focused like 'ensuring there are answers to users' questions'. 

This is pedantic and I don't actually have a strong feeling here (mainly because I'd personally just work across both areas if I was able). But this is further reflected in a later edit where you reiterate this point and attach it to W+D there: 

```
- **Documentation** Working with Website & Docs, the content team ensures docs are maintained to a high standard, and users can easily find answers to their questions before resorting to support.
```

This is also ties in slightly to the conversation in Slack where it was suggested an individual marketing owner should own their respective area of the docs. We're due to discuss that in Sprint meeting, but it strikes me there's a philosophical shift there: viewing the docs as a marketing responsibility, vs as an extension of the product. 

Again, no super strong feelings here but I'm curious if this matches to your intention.

Reply:
Yeah, I guess I'd just say that an obvious way to solve `2. Ensuring users can autonomously find answers to their questions` as an objective would be to improve the site search further, rebuild the docs structure, change the way the API docs are built, etc. 

I'd suggest changing this to make it clear it's owned jointly, or rephrase. 

```suggestion
Marketing & Content exists to drive organic, word-of-mouth growth by:

1. Producing world-class top-of-funnel content for our ICP
2. Work with Website and Docs to ensure the information users need is available, high-quality and can be easily found 

```

Reply:
For elegance, I'll drop the "Work with Website and Docs" bit. It's included further down and we know it's a shared objective.